ad7768-1fmcz/ad77681.c

```diff
@@ -211,30 +235,76 @@ uint8_t ad77681_get_rx_buf_len(struct ad77681_dev *dev)
 }
 
 /**
+ * Helper function to get the number of SPI bytes for INTERRUPT ADC DATA READ
+ * @param dev - The device structure.
+ * @return frame_bytes - the number of rx bytes
+ */
+uint8_t ad77681_get_frame_bytes(struct ad77681_dev *dev)
```

Comment:
I think this function is used only local. So, probably it should be removed from header, and make it static.